DataFormats/Headers/src/DataHeader.cxx

```diff
@@ -27,14 +27,15 @@ const o2::Header::SerializationMethod o2::Header::gSerializationMethodFlatBuf("F
 //__________________________________________________________________________________________________
 //possible data origins
 const o2::Header::DataOrigin o2::Header::gDataOriginAny    ("***");
-const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("   ");
+const o2::Header::DataOrigin o2::Header::gDataOriginInvalid("INV");
```

Comment:
keeping in mind what we always said about null termination here - I like it less and less... Maybe it would qualify for a rediscussion?